### PR TITLE
allow ManagedCluster creation with taint.timeAdded specified

### PR DESCRIPTION
**Pull request by elgnay**
Signed-off-by: Yang Le <yangle@redhat.com>

### CODE REVIEW

File: pkg/webhook/cluster/mutating_webhook.go [hunk 1]
```diff
@@ -71,13 +71,15 @@ func (a *ManagedClusterMutatingAdmissionHook) Admit(req *admissionv1beta1.Admiss
...
 	var jsonPatches []jsonPatchOperation
+	var warnings []string
 
 	// set timeAdded of taint if it is nil and reset it if it is modified
 	taintJsonPatches, status := a.processTaints(managedCluster, req.OldObject.Raw)
 	if !status.Allowed {
 		return status
 	}
 	jsonPatches = append(jsonPatches, taintJsonPatches...)
+	warnings = append(warnings, status.Warnings...)
```
Comment (qiujian16) [inline pkg/webhook/cluster/mutating_webhook.go]: why we need to do append here?

Comment (elgnay) [reply pkg/webhook/cluster/mutating_webhook.go]: Redundant code removed.

**Pull request merged**